mls-rs/src/extension.rs

```diff
@@ -9,6 +9,10 @@ pub(crate) use built_in::*;
 /// Default extension types required by the MLS RFC.
 pub mod built_in;
 
+/// Draft extension types.
+#[cfg(feature = "last_resort_key_package_ext")]
+pub mod draft;
```

Comment:
Why `draft`?

Reply:
I am happy to call the module anything you like — I thought the extensions in [the RFC](https://datatracker.ietf.org/doc/html/draft-ietf-mls-extensions-04) are still draft extensions? Or rather that the RFC is a draft?

I called the module `proposals` at some point. I don't know if that's better?

Reply:
Ah I see. The RFC is a draft, yes. I thought this meant that the implementation is a draft, and got confused. Maybe something like "non-default" since the comment above `built_in` mentions "default"?

Why `proposals` is this is about key packages?

Reply:
> Ah I see. The RFC is a draft, yes. I thought this meant that the implementation is a draft, and got confused. Maybe something like "non-default" since the comment above `built_in` mentions "default"?

I'm not sure the distinction is unnecessary. More precisely, the extensions in the `built_in` module are the ones that have _Recommended: yes_ in [17.3. ](https://www.rfc-editor.org/rfc/rfc9420.html#section-17.3)[MLS Extension Types](https://www.rfc-editor.org/rfc/rfc9420.html#name-mls-extension-types).

The extensions in [The Messaging Layer Security (MLS) Extensions](https://datatracker.ietf.org/doc/html/draft-ietf-mls-extensions-04) are actually also recommended, but they don't have a corresponding RFC yet.

Since you already have a `pub(crate) use built_in::*`, I would actually propose to just get rid of the `built_in` module. Or perhaps it could be renamed to `recommended` if you like? Then we could add the last resort extension there.
 
> Why `proposals` is this is about key packages?

Here I meant "proposal" in the sense of "proposed extensions", but I see that it's confusing when the protocol itself uses this term :smile:

Reply:
There is a difference between "recommended" and "built-in" which I think was renamed to "default" in the RFC. See [here](https://www.rfc-editor.org/rfc/rfc9420.html#section-7.2-4). Default extensions MUST be supported by all MLS clients. Recommended extensions may or may not be supported but they're OK to support.

The extensions in the `built_in` module are "default" ones. If there is a "recommended" extensions in the Extensions RFC, then it would not be "default" and so not in `built_in`

Reply:
Okay, would you like me to rename the module to `recommended`? Any name is fine with me :smile:

Reply:
Yeah, I just found "draft" misleading, sounds like "not fully implemented". Recommended is fine, I don't mind as long as it's not very confusing.

Reply:
Super, I renamed it to `recommended` now. I also added a `pub use` import like for `built_in` — I feel this is helpful since it removes the need for others to think about this detail.

Reply:
Awesome